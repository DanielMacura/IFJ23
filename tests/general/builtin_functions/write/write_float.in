write(42.1)
write("\n")
