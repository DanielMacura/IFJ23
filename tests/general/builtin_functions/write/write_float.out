0x1.50ccccccccccdp+5
